doc/nrf/ug_nrf53.rst

```diff
@@ -4,6 +4,7 @@ Working with nRF53 Series
 #########################
 
 The |NCS| provides support for developing on the nRF5340 System in Package (SiP) using the nRF5340 DK and the Nordic Thingy:53.
+Thingy:53 is currently an unreleased product that will be released in the near future.
```

Comment:
1) "currently" is not very anchored in time. Maybe:
"As of the |NCS| v1.7.0 release, Thingy:53 is an unreleased product that will be released in the near future."
In such case this PR needs to be pointed to the "update docs to v1.7.0 for relase" PR (#5507 )
2) I feel this is a temporary note, not a part of the user guides. So maybe place this in a note and use the same text for both? (shortcuts.txt)